### PR TITLE
Fallback to default theme resources

### PR DESCRIPTION
**Pull request by blakeembrey**

### CODE REVIEW

**Comment by felixfbecker** [comment]
Does anything prevent this from getting merged?

**Comment by blakeembrey** [comment]
I don't think so, but knowing more people have tested it would help. I'm not familiar with the codebase, and I also couldn't find how this feature could have been broken.

File: src/lib/output/renderer.ts [hunk 1]
```diff
@@ -199,14 +199,7 @@ export class Renderer extends ChildableComponent<Application, RendererComponent>
                 }
             }
 
-            var filename = Path.join(path, 'theme.js');
```
Comment (Dema) [inline src/lib/output/renderer.ts]: Could we bring this back. As I understand this breaks typedoc-markdown-theme. It has some code in theme.js and now it's being ignored.

Comment (blakeembrey) [inline src/lib/output/renderer.ts]: Can you share how that's possible? The code here ran exactly the same regardless of this check.

**Pull request merged**